crates/saya/core/src/verifier/starknet.rs

```diff
@@ -0,0 +1,51 @@
+use starknet::{
+    accounts::{Account, Call, ExecutionEncoding, SingleOwnerAccount},
+    core::{
+        types::{BlockId, BlockTag, FieldElement},
+        utils::get_selector_from_name,
+    },
+    providers::{jsonrpc::HttpTransport, JsonRpcClient},
+    signers::{LocalWallet, SigningKey},
+};
+use url::Url;
+
+// will need to be read from the environment for chains other than sepoia
```

Comment:
Yes, good point. We need to figure out the best way to configure this.

1. If the user wants to run it's own saya service, reading from env it's the best or we can actually pass them using the CLI.
2. In the case of SLOT, we should have a default account, and the user approve spending and pass an account address from which saya will withdraw the fees.

Reply:
Could you clarify what do you mean by slot?

Reply:
> Could you clarify what do you mean by slot?

Slot is a service where Katana, Torii and later Saya could be deployed. So the users does not have direct control on the server. But somehow it's out of the PR, I was making the comment to keep awareness on that. :+1: